README.md

```diff
@@ -1,18 +1,55 @@
-# scout-alexa
+# Scout Alexa Skill
 
-This is a lambda function that drives the Alexa skill for Scout.  
+[![Build Status](https://travis-ci.org/MozScout/scout-alexa.svg?branch=master)](https://travis-ci.org/MozScout/scout-alexa)
+[![Coverage Status](https://coveralls.io/repos/github/MozScout/scout-alexa/badge.svg?branch=master)](https://coveralls.io/github/BScong/scout-alexa?branch=master)
 
-### Cross Compiling the natural packages
-In order to run this on Amazon, some of the node packages need to be cross-compiled on Amazon linux to work properly.  Here is a set of steps to do this:
-* Create an EC2 instance of Amazon Linux.  [Create an EC2 instance of Amazon Linus](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html)
+This is a lambda function that drives the Alexa skill for Scout.
+
+## Setting up
+
+Requirements: [scout-ua](https://github.com/MozScout/scout-ua), Auth Lamdba Functions and Auth API Gateways are deployed.
+
+### AWS Lambda
+
+Create an AWS Lambda Function on [AWS Lambda](https://console.aws.amazon.com/lambda/).
+
+Add the following environment variables to your Lambda function:
+
+* `JWOT_TOKEN`: JWOT token to access API (scout-ua)
+* `SCOUT_ADDR`: API hostname for scout-ua (`scout-ua.herokuapp.com` for instance)
```

Comment:
I think remove the example there and just put yourserver.com or something since that is an actual server of ours.